src/extensions/migration-utils/is-outdated.js

```diff
@@ -0,0 +1,92 @@
+const TARGET_NODE_VERSION = process.env.TARGET_NODE_VERSION;
+const TARGET_HW_DEVICE_FIRMWARE = process.env.TARGET_HW_DEVICE_FIRMWARE;
+
+export const updateIsOutdatedColumn = async (knex) => {
+	if (!TARGET_NODE_VERSION || !TARGET_HW_DEVICE_FIRMWARE) {
+		throw new Error('TARGET_NODE_VERSION and TARGET_HW_DEVICE_FIRMWARE must be set');
+	}
+
+	await knex.transaction(async (trx) => {
```

Comment:
Seems this could be just `knex.raw()`? The transaction doesn't really do anything here. Same below.